KubeArmor/enforcer/bpflsm/enforcer.go

```diff
@@ -341,9 +340,13 @@ func (be *BPFEnforcer) TraceEvents() {
 			log.Source = string(bytes.Trim(event.Data.Source[:], "\x00"))
 			log.Resource = string(bytes.Trim(event.Data.Path[:], "\x00"))
 			log.Enforcer = "BPFLSM"
```

Comment:
We should move this line to after Switch Case as well. And see if we can handle anything else in a common way to deduplicate things.